### ISSUE TITLE
Include video of game somewhere in the site

### ISSUE THREAD

**Issue by ScalaWilliam** [ScalaWilliam/ActionFPS#308]
Probably in /play/, so newcomers know what to look at.

**Comment by Paul-Michaud** [comment]
Is this still relevant ?

**Comment by ScalaWilliam** [comment]
Yes, I'll assign to myself since I'm thinking of simplifying the homepage and putting it there :-)

I'll use the GIF you had made (thanks!).
